tests/frontend-storage/src/frontend/test-templates/DownloadTests.ts

```diff
@@ -0,0 +1,100 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ObjectReference } from "@itwin/object-storage-core";
+
+import {
+  assertArrayBuffer,
+  assertReadableStream,
+  stringToArrayBuffer,
+} from "../Helpers";
+
+import { TestProps } from "./Interfaces";
+
+async function setupDownload(testName: string, test: TestProps) {
```

Comment:
Would be nice to have an internal interface defined for this set of properties and function return type - it is a bit hard to read what setup is.